addons/mail/wizard/mail_compose_message.py

```diff
@@ -169,6 +169,10 @@ def _partner_ids_domain(self):
         'Keep Message Copy',
         compute="_compute_auto_delete_keep_log", readonly=False, store=True,
         help='Keep a copy of the email content if emails are removed (mass mailing only)')
+    bypass_blacklist = fields.Boolean('Include Blacklist',
```

Comment:
``include_excluded``? That way if we go for a positive meaning (include excluded emails), we align field name with its meaning.